content/cpp/concepts/math-functions/terms/fmax/fmax.md

```diff
@@ -0,0 +1,63 @@
+---
+Title: 'fmax()'
+Description: 'Returns the larger of two arguments.'
+Subjects:
+  - 'Computer Science'
+Tags:
+  - 'Functions'
+  - 'Arithmetic'
+  - 'Methods'
+CatalogContent:
+  - 'learn-c-plus-plus'
+  - 'paths/computer-science'
+---
+
+The **`fmax()`** function returns the larger of two arguments.
+
+## Syntax
+
+```pseudo
+fmax(x, y)
+```
+
+The parameters `x` and `y` are usually a numeric [data type](https://www.codecademy.com/resources/docs/cpp/data-types) ranging from `-INFINITY` (negative) to `INFINITY` (positive). If one of the parameters is `NAN`, then the other parameter is returned.
```

Comment:
- Can we change `NAN` (all caps) to `NaN`
- The reference to positive `INFINITY` works, but perhaps we can change it to "from negative `INFINITY` to positive `INFINITY`."

Reply:
@yangc95 What about something like this? "from negative `-INFINITY` to positive `INFINITY`"?